src/ClientHandle.cpp

```diff
@@ -1924,7 +1924,7 @@ void cClientHandle::Tick(float a_Dt)
 		{
 			m_PingID++;
 			m_PingStartTime = std::chrono::steady_clock::now();
-			m_Protocol->SendKeepAlive(m_PingID);
+			m_Protocol->SendKeepAlive(static_cast<UInt32>(m_PingID));
```

Comment:
Curious. According to [the Minecraft protocol](http://wiki.vg/Protocol#Data_types) the VarInt type should be that same as [the Protobuf Varint](https://developers.google.com/protocol-buffers/docs/encoding#varints). However looking at [the encoding code](https://github.com/cuberite/cuberite/blob/6e4122e/src/ByteBuffer.cpp#L743-L759) it seems that the protocol doesn't employ [the ZigZag encoding](https://developers.google.com/protocol-buffers/docs/encoding#signed-integers). So it is the two's complement then, right?